README.md

```diff
@@ -66,34 +66,40 @@ Install the ``python3-routes`` package:
 
 These instructions are meant for developers who are compiling the code for
 development and testing. To build binaries that are suitable for installation
-we recommend that you build .deb or .rpm packages, or refer to ``ceph.spec.in``
-or ``debian/rules`` to see which configuration options are specified for
-production builds.
+we recommend that you build `.deb` or `.rpm` packages, or refer to
+``ceph.spec.in`` or ``debian/rules`` to see which configuration options are
+specified for production builds.
 
-Build instructions:
+To build ceph, make sure that you are in the top-level `/ceph` directory, and
+run the following commands:
 
 	./do_cmake.sh
 	cd build
 	ninja
 
-``do_cmake.sh`` defaults to creating a debug build of Ceph that can be up to 5x
-slower with some workloads. Pass ``-DCMAKE_BUILD_TYPE=RelWithDebInfo`` to
+``do_cmake.sh`` by default creates a "debug build" of Ceph, which can be up to
+5x slower with some workloads. Pass ``-DCMAKE_BUILD_TYPE=RelWithDebInfo`` to
```

Comment:
Pet peeve:  what are the units of slowness?  

which can run at as little as 20% the speed of a non-debug build

Reply:
I've rewritten this a bit, and s/release/build/, because I don't think that "release" is correct. This is a build, and a build of what is perhaps a release candidate, but it (as far as I can tell) is not a release.